merlin/dag/selector.py

```diff
@@ -46,6 +46,12 @@ def __init__(
         self._tags = tags if tags is not None else []
         self.subgroups = subgroups if subgroups is not None else []
 
+        self.all = names == "*"
```

Comment:
names is annotated to be a `List[str]` and the docstring says it could also be a list of tuples of str. Should this check for those possible inputs as well?

Reply:
supporting `["*"]` as a value that activates this functionality as well as `"*"`?

Reply:
For consistency, we could make `["*"]` the syntax for it. That opens up the possibility of something like `["*", "column1"]` (redundant) or `[("*", "column1")]` (non-sensical), which we'd then have to handle.

Reply:
(Tuples mean "group these columns together," which some operators use to understand how to apply their transforms. Categorify's "joint" and "combo" encoding modes are examples of cases where it's important to understand how columns should be grouped together.)

Reply:
(For the record, I did not design this syntax or write the operators that rely on it, that was all before my time. ColumnSelector maintained it in the interest of avoiding breaking changes and, in all honesty, because no coherent alternative proposals have been put forward.)